setup.py

```diff
@@ -363,10 +364,11 @@ def check_file_at_path(path2file):
 
 packages = find_packages(include=["numba", "numba.*"])
 
-build_requires = ['numpy >={}'.format(min_numpy_build_version)]
+build_requires = ['numpy >={}<{}'.format(min_numpy_build_version,
+                                         max_numpy_run_version)]
 install_requires = [
     'llvmlite >={},<{}'.format(min_llvmlite_version, max_llvmlite_version),
-    'numpy >={}'.format(min_numpy_run_version),
+    'numpy >={}<{}'.format(min_numpy_run_version, max_numpy_run_version),
```

Comment:
```suggestion
    'numpy >={},<{}'.format(min_numpy_run_version, max_numpy_run_version),
```
same question RE comma